pkg/virt-launcher/virtwrap/network/ndp/connection.go

```diff
@@ -146,18 +146,18 @@ func (l *NDPConnection) GetFD() (*os.File, error) {
 	return l.rawConn.File()
 }
 
-func (l *NDPConnection) ReadFrom() (ndp.Message, *ipv6.ControlMessage, error) {
+func (l *NDPConnection) ReadFrom() (ndp.Message, error) {
```

Comment:
When we will finish the review process, please squash this commit with the one actually introduced the CM. Meanwhile, you can mention in the commit message "this commit is going to be squashed with ..."